### ISSUE TITLE
Feature Idea: 

### ISSUE THREAD

**Issue by arturoovf** [mattermost/focalboard#3184]
## Summary

Support for 
Calendar Feed or CalDAV 

## How important this is to me and why

Importance: High

I think the feature to have a calendar feed, to be able to sync the calendar of focal board to third party calendars is fundamental.  This way the cards with a date on it would appear in the calendar of your devices automatically

Use cases:
1. Integrate with system calendar, along side your other calendars in one single view
2. Add a card with a date on it and receive notifications in all your devices automatically

Any examples of good implementations of this capability
Trello does this with a calendar power up, exports a calendar feed that calendar on the Mac/iOS/ipadOS/WatchOS can add and have it show on all devices

**Comment by broskees** [comment]
This is huge. Another example of a good great implementation of capability, I think the best in the market, is [TickTick](https://ticktick.com/?language=en_US). The calendar view allows you to actually time-box your day with tasks. Then you can simply add the .ical link to your current calendar app.

**Comment by gignsky** [comment]
This would be absolutely fantastic! I would use this feature ALL the time.

**Comment by photonzoo** [comment]
+1 I second this feature request: very few Kanban board solutions actually support users importing deadlines for tasks and activities into their personal calendars. Doing so, Focalboard would provide an extremely useful differentiator allow end users to more easily keep track of their schedules. Among other advantages, it would permit end users to set themselves calendar alerts according to their personal situation, having a full view of commitments. 
Focalboard implementing a CalDAV server with individual feeds that automatically updated each user's personal calendar would be strongly preferable over a solution involving iCal invites, as the latter required tedious, repetitive manual action on the end users' side, risked running into potential issues with entries not getting properly updated and – if sent by, e.g., email – unnecessarily clutter their inboxes.

**Comment by arturoovf** [comment]
> +1 I second this feature request: very few Kanban board solutions actually support users importing deadlines for tasks and activities into their personal calendars. Doing so, Focalboard would provide an extremely useful differentiator from other solutions, allowing end users to more easily keep track of their schedules. Among other advantages, it permitted end users to set themselves calendar alerts according to their personal situation, having a full view of commitments.
> Focalboard implementing a CalDAV server with individual feeds that automatically updated each user's personal calendar would be strongly preferable over a solution involving iCal invites, as the latter required tedious, repetitive manual action on the end users' side, risked running into potential issues with entries not getting properly updated and – if sent by, e.g., email – unnecessarily clutter their inboxes.

Indeed CalDAV server with individual feeds would be absolutely amazing!

**Comment by arturoovf** [comment]
gently bump

**Comment by albert118** [comment]
+ 1 to bump

Having used the Trello powerup, this is now a must in my workflows. Services like Jira are really shown up by this integration capability. Using Jira on a daily this definitely something I notice as a heavy Google calendar user.

I would even be happy with something as simple as my cards with due dates being imported as tasks without a timestamp in Google calendar. Although a timestamp would be "nice-to-have".

The closet similar implementation (aside from Trello's) is using [Micrsoft To Do and Zapier](https://zapier.com/apps/google-calendar/integrations/microsoft-todo) but this is a pain!

**Comment by q-wertz** [comment]
You can also compare to the features of [Nextcloud Deck](https://github.com/nextcloud/deck) app. The tasks are fully integrated into their Task app and therefore be used by any CalDav compliant client.

**Comment by q-wertz** [comment]
You can also compare to the features of [Nextcloud Deck](https://github.com/nextcloud/deck) app. The tasks are fully integrated into their Task app and therefore be used by any CalDav compliant client.

**Comment by q-wertz** [comment]
You can also compare to the features of [Nextcloud Deck](https://github.com/nextcloud/deck) app. The tasks are fully integrated into their Task app and therefore be used by any CalDav compliant client.

**Comment by chaeyan** [comment]
bump!

**Comment by update-freak** [comment]
It would be great if focalboard will support CalDAV.
aCalendar+ (Android) <Sync with DAVx5> Focalboard (NAS) <Outlook Caldav Sychoronizer> Outlook

For me its important that the priority of aCalendar+,  Focalboard, Outlook will use the same values.

**Comment by dimonic** [comment]
Yes to this, I am trying to settle on a Single Authoritative Calendar, and webdav would make that so simple.
